docker/docker-compose.yml

```diff
@@ -10,3 +10,11 @@ services:
       MONGO_INITDB_ROOT_PASSWORD: example
     volumes:
       - ./data:/data/db
+
+  scratchy:
+    image: scratchy-server:latest
+    ports:
+      - "5000:5000"
+    links:
+      - "mongodb"
+
```

Comment:
Missing configuration file, the scratchy server won't work without it as the default configuration will try to reach the database at `localhost` and not `mongodb`